app/config/config.go

```diff
@@ -35,6 +39,21 @@ type Conf struct {
 		FilesLocation   string             `yaml:"files_location"`
 		RSSLocation     string             `yaml:"rss_location"`
 	} `yaml:"youtube"`
+
+	Telegram struct {
```

Comment:
probably both telegram and twitter should be under System, maybe even System.Notifications

Reply:
Done